packages/calypso-e2e/src/lib/components/isolated-block-editor-component.ts

```diff
@@ -0,0 +1,56 @@
+import { Page, ElementHandle } from 'playwright';
+
+const selectors = {
+	blockInserterToggle: 'button[aria-label="Toggle block inserter"]',
+	blockInserterSearch: 'input[placeholder="Search"]',
+	blockInserterResultItem: '.block-editor-block-types-list__list-item',
+
+	// Publish
+	postButton: 'a:text("Post")',
+};
+/**
+ * Represents an instance of the WPCOM's Gutenberg editor page.
+ */
+export class IsolatedBlockEditorComponent {
```

Comment:
I took this name from https://github.com/Automattic/isolated-block-editor/ which appears to be the name for anytime Gutenberg editor is used but not within the context of a WordPress.com site. So in this case, a P2 frontend editor matches this description.